src/main/java/com/google/cloud/tools/appengine/api/devserver/DefaultRunConfiguration.java

```diff
@@ -17,15 +17,15 @@
 package com.google.cloud.tools.appengine.api.devserver;
 
 import java.io.File;
+import java.nio.file.Path;
 import java.util.List;
 
 /**
  * Plain Java bean implementation of {@link RunConfiguration}.
  */
 public class DefaultRunConfiguration implements RunConfiguration {
 
-  private List<File> appYamls;
-  private List<File> services;
+  private List<Path> services;
```

Comment:
We don't want to use Path just yet as it messed up the gradle configuration unfortunately.